env.example

```diff
@@ -76,5 +76,4 @@ SHOW_DISCONNECT_MESSAGE=true
 #                                                                             #
 ###############################################################################
 
-USE_IMGUR_FOR_IMAGES=true
```

Comment:
I assume this is leftover from v1?

Reply:
Yes, we never required imgur, but at this point making imgur optional just makes for more unnecessary overhead.